WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift

```diff
@@ -254,6 +233,60 @@ class JetpackScanCoordinator {
         sections = [JetpackThreatSection(title: nil, date: Date(), threats: threats)]
     }
 
+    // MARK: - Private: Network Handlers
+    private func refreshDidSucceed(with scanObj: JetpackScan) {
+        scan = scanObj
+        view.render()
+
+
+        togglePolling()
+    }
+
+    private func refreshDidFail(with error: Error? = nil) {
+        let appDelegate = WordPressAppDelegate.shared
+
+        guard
+            let connectionAvailable = appDelegate?.connectionAvailable, connectionAvailable == true
+        else {
+            view.showNoConnectionError()
+            actionButtonState = .tryAgain
+
+            return
+        }
+
+        view.showGenericError()
+        actionButtonState = .contactSupport
+    }
+
+    private func scanDidChange(from: JetpackScan?, to: JetpackScan?) {
+        let fromState = from?.state ?? .unknown
+        let toState = to?.state ?? .unknown
+
+        // Trigger scan finished alert
+        guard fromState == .scanning, toState == .idle else {
+            return
+        }
+
+        let threatCount = threats?.count ?? 0
+
+        let message: String
+
+        switch threatCount {
+            case 0:
+                message = Strings.scanNotice.message
+
+            case 1:
+                message = Strings.scanNotice.messageSingleThreatFound
+
+            default:
+                message = String(format: Strings.scanNotice.messageThreatsFound, threatCount)
+        }
+
+        let notice = Notice(title: Strings.scanNotice.title, message: message)
+
+        ActionDispatcher.dispatch(NoticeAction.post(notice))
```

Comment:
You can use that and remove the flux import: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Extensions/UIViewController+Notice.swift

Reply:
Done in: https://github.com/wordpress-mobile/WordPress-iOS/pull/15921/commits/0e7f8f87809188719c657729e1ead60cb69df4d9